### PR TITLE
Update Role#member_of_congress? and update some factories

### PR DESCRIPTION
**Pull request by handlers**
So I made a couple of changes to `spec_helper.rb`, which leads to some red tests that I will fix when we get to the Travis implementation. Mostly I am doing a PR to see if people have any ideas for other edge cases to test in [role_spec.rb](https://github.com/sunlightlabs/opencongress/blob/4fdb01077549af789ad97a6aa5cfbf189173b490/spec/models/role_spec.rb).

### CODE REVIEW

File: app/models/role.rb [hunk 1]
```diff
@@ -64,8 +64,11 @@ def chamber
   #
   # @param congress [Integer] congress number
   # @return [Boolean] true if member, false otherwise
-  def member_of_congress?(congress)
-    NthCongress.where('number = ? AND (start_date >= ? OR end_date >= ?)', congress.nil? ? NthCongress.current : NthCongress.find(congress), self.startdate, self.enddate).any?
+  def member_of_congress?(congress=Settings.default_congress)
```
Comment (crdunwel) [inline app/models/role.rb]: Yes :+1:

**Comment by crdunwel** [comment]
Me gusta esto
